source/Releases/Release-Foxy-Fitzroy.rst

```diff
@@ -288,6 +288,36 @@ There are copies in `example_interfaces <https://index.ros.org/p/example_interfa
 - ``std_msgs/msg/UInt8``
 - ``std_msgs/msg/UInt8MultiArray``
 
+sros2
+^^^^^
+
+Use of security enclaves
+""""""""""""""""""""""""
+
+As of Foxy, domain participants are not mapped directly to ROS nodes anymore.
+The concept of security enclave was introduced, an enclave is a process or group of processes that will share the same identity and access control rules.
+Each process using ROS Security is part of an enclave.
+
+This means that security artifacts are **not** retrieved based on the node name anymore but based on the Security enclave name.
```

Comment:
Assuming you like my edit above, we can probably get rid of this sentence.

```suggestion
```

Reply:
My goal here was to highlight what it actually changes for users. The paragraph above is generic but it's unclear from the outside what concrete change a user need to perform to migrate a secured system from Eloquent to Foxy. E.g. it doesnt seem too far-fatched for a user to assume that by default the nodes would use an enclave matching the node name

Reply:
Okay fair enough.